src/components/LoginRegisterForm/PassInputField.jsx

```diff
@@ -1,7 +1,7 @@
 import React, {useState}from'react';
 import LoginProgress from './LoginProgress';
 
-import style from './LoginRegisterForm.module.css';
+import style from './LoginRegisterForm.module.scss';
```

Comment:
de ce e fisierul de scss module in Componenta de LoginRegisterForm cand ai un folder de sass i ncare sa tii fisierele de modules.sass?